plugin.go

```diff
@@ -200,6 +202,20 @@ func commandBuild(build Build) *exec.Cmd {
 		args = append(args, "--build-arg", arg)
 	}
 
+	labelSchema := []string{
+		fmt.Sprintf("build-date=%s", time.Now().Format(time.RFC3339)),
+		fmt.Sprintf("vcs-ref=%s", build.Name),
+		fmt.Sprintf("vcs-url=%s", build.Remote),
+	}
+
+	if len(build.LabelSchema) > 0 {
+		labelSchema = append(labelSchema, build.LabelSchema...)
+	}
+
+	for _, label := range labelSchema {
+		args = append(args, "--label", fmt.Sprintf("org.label-schema.%s", label))
```

Comment:
should we automatically prefix the `org.label-schema` for the user?  This plugin generally forces explicit values instead of implicit (for example, the repo field is not automatically prefixed with the registry hostname). I don't have a strong opinion either way, so wanted to pose the question. If you think this is the best approach, then no changes required :)